### DISCUSSION TITLE
Couldn't find any versions for "url-slug" that matches "^3.0.3"

### DISCUSSION THREAD

**Discussion by 5idu** [loopbackio/loopback-next#8307] [Q&A] [answered]
when install loopback4 cli, the following error occurs: 
```sh
npm i -g @loopback/cli
```
`Couldn't find any versions for "url-slug" that matches "^3.0.3"`. 

In fact，the `url-slug` latest version is "3.0.2". 
So should you update the dependency version of `url-slug` ?

**Comment by achrinza** [comment]
Just tested with the latest version, it seems to be working fine. Perhaps it was a momentary issue with a deep dependency, could you try it again?

Or you may try using the CLI with this command instead:

```sh
$ npx @loopback/cli
```

**Reply by 5idu** [reply]
<img width="1014" alt="image" src="https://user-images.githubusercontent.com/26264096/154445860-e38e0cb3-a21b-4f54-b8c6-06b1f83d56e3.png">

still not work!

**Comment by 5idu** [comment, marked as answer]
I eventually figured out that it was an issue with the `npm registry proxy`, this discussion will be closed.
